po/vi.po

```diff
@@ -7097,21 +7101,24 @@ msgstr ""
 
 #: src/welcome-day-3.md
 msgid "Welcome to Day 3"
-msgstr ""
+msgstr "Chào mừng tới Ngày 3"
 
 #: src/welcome-day-3.md
 msgid "Today, we will cover:"
-msgstr ""
+msgstr "Chủ đề chúng ta sẽ học là:"
```

Comment:
"Chủ đề hôm nay chúng ta sẽ học là" or "Chủ đề chúng ta sẽ học hôm nay là:" might be "more" closely translated. If you don't like it I can still resolve it btw.

Reply:
Thank you, I've also changed the sentence to make it more natural.